code/datums/action.dm

```diff
@@ -162,6 +162,14 @@
 /datum/action/item_action/print_report
 	name = "Print Report"
 
+/datum/action/item_action/print_forensic_report
+	name = "Print Report"
+	button_icon_state = "scanner_print"
+	use_itemicon = FALSE
+
+/datum/action/item_action/clear_records
+	name = "Clear Scanner Records"
+
```

Comment:
Doesn't this need its own icon? Like a picture of the scanner with an 'X' or something?

Reply:
I tried that, but I couldn't work out a way to make it look good.
I figured it'd be fine with just the default icon, since the 'Print Scanner Report' icon is different now.